beam/beam/handling_unit.py

```diff
@@ -19,7 +19,10 @@ def generate_handling_units(doc, method=None):
 		return doc
 
 	for row in doc.items:
-		if not frappe.get_value("Item", row.item_code, "is_stock_item"):
+		if not (
+			frappe.get_value("Item", row.item_code, "is_stock_item")
+			and frappe.get_value("Item", row.item_code, "enable_handling_unit")

```

Comment:
get_value can handle multi fields return
frappe.get_value("Item", row.item_code, ["is_stock_item","enable_handling_unit"])

Reply:
Code is Updated